clpy/core/include/clpy/carray.clh

```diff
@@ -1,5 +1,7 @@
 #pragma once
+#ifdef __CLPY_ENABLE_CL_KHR_FP16
```

Comment:
Why didn't you just remove cl_khr_fp16 for all environments but disable it only if it doesn't support it?

Do you have any definite disadvantages or problems when you treat half as int16 on a cl_khr_fp16 machine? If yes, please leave comments on the code, otherwise (yet) please make the code simple.

Reply:
I think that it's better in performance to use `half` implemented as hardware. If we will support all `half` operations, the performance given from native `half` in supported environments is more important.

Reply:
> If we will support all half operations

We have not proven the ability neither decided to do that, must continue to discuss on #265 .

Then, until to decide it, the performance without native half support is not important.
Adding `cl_khr_fp16` is too superfluous to just disable it #264 .

Reply:
> must continue to discuss on #265 .

Sure.

> Adding `cl_khr_fp16` is now too superfluous to just disable it #264 .

So, you want to just remove `cl_khr_fp16` whether the extension is available or not, right?